q2_annotate/busco/busco.py

```diff
@@ -140,6 +154,24 @@ def _evaluate_busco(
     common_args = _process_common_input_params(
         processing_func=_parse_busco_params, params=kwargs
     )
+    if isinstance(mags, MultiMAGSequencesDirFmt):
+        busco_results = _busco_helper(mags, common_args, additional_metrics)
```

Comment:
Why does this need to be called twice, depending on the if-statement result? You should pull it out of this block and just modify the results accordingly. Then, there should be only a single `return` statement at the very end which will return `busco_results`. This way the flow is a bit more clear.